### ISSUE TITLE
Custom modal dialog information not accessible

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#96603]
From @isidorn 

Get a modal dialog in the web
Notice how you can nicely focus any of the buttons, however the content of the dialog message is never read out, thus the dialog is not accessible.

How this should work:

Dialog should have an aria label and have focus
It shuld use active descendent to let the screen reader know that focus is passed to one of the buttons

**Comment by sbatten** [comment]
@bpasero I am using the button widgets, are notifications not accessible either or are they doing something special?

**Comment by bpasero** [comment]
Can you clarify what you mean exactly? Notifications announce themselves as ARIA alert when they open: https://github.com/microsoft/vscode/blob/a716714a88891cad69c0753fb95923870df295f5/src/vs/workbench/browser/parts/notifications/notificationsAlerts.ts#L14

**Comment by sbatten** [comment]
I misread the initial issue, I thought the bug was that the buttons were not read, not the actual message.

**Comment by isidorn** [comment]
@sbatten Thanks for fixing, let's add a bug label so we get this verified.
